experimental/generation/runbot/Startup.cs

```diff
@@ -51,6 +54,16 @@ public void ConfigureServices(IServiceCollection services)
             ComponentRegistration.Add(new LanguageGenerationComponentRegistration());
             ComponentRegistration.Add(new QnAMakerComponentRegistration());
             ComponentRegistration.Add(new LuisComponentRegistration());
+            ComponentRegistration.Add(new DeclarativeComponentRegistrationBridge<DynamicListBotComponent>());
+            ComponentRegistration.Add(new DeclarativeComponentRegistrationBridge<LuisBotComponent>());
+/*
+            new DialogsBotComponent().ConfigureServices(services, this._configuration);
+            new DeclarativeBotComponent().ConfigureServices(services, this._configuration);
+            new AdaptiveBotComponent().ConfigureServices(services, this._configuration);
+            new LanguageGenerationBotComponent().ConfigureServices(services, this._configuration);
+            new QnAMakerBotComponent().ConfigureServices(services, this._configuration);
+            new LuisBotComponent().ConfigureServices(services, this._configuration);
+            new DynamicListBotComponent().ConfigureServices(services, this._configuration);*/
 
```

Comment:
When I use the new-version registration codes, the bot could not be correctly started in the emulator. Do I miss something for this._configuration?